make/langtools/src/classes/build/tools/symbolgenerator/CreateSymbols.java

```diff
@@ -959,6 +962,21 @@ private void addAttributes(ClassHeaderDescription header,
             attributes.put(Attribute.NestMembers,
                            new NestMembers_attribute(attributeString, nestMembers));
         }
+        if (header.recordComponents != null && !header.recordComponents.isEmpty()) {
```

Comment:
I am not sure of the exact logic here, but it is perfectly fine for a record attribute to contain zero components, and for the class to still be considered a "record class". But maybe that is not all that significant here? I just want to call it out so that it is considered.

Reply:
Ah, right - fixed in:
https://github.com/openjdk/jdk/pull/1480/commits/e1ec2b7ff49e1307a348ad58b12e5ed39ebe7224

Thanks for the comment!